app/src/main/java/ar/com/wolox/android/cookbook/common/di/AppModule.kt

```diff
@@ -21,4 +28,25 @@ abstract class AppModule {
 
     @ContributesAndroidInjector
     abstract fun googleLoginFragment(): GoogleLoginRecipeFragment
+
+    @ContributesAndroidInjector
+    abstract fun navigationActivity(): NavigationActivity
```

Comment:
Adding all of these classes to the `AppModule` pollutes it. To separate responsibility, we should do what I did in the **data-sync** branch.